sdks/java/io/hadoop-common/pom.xml

```diff
@@ -32,6 +32,17 @@
 
   <dependencies>
     <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-sdks-java-core</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
It was like that in `hdfs-io`, and if users need to use a different jackson version they should do so explicitly - the Spark runner does that.